### PR TITLE
Deprecate Flash (ActionScript) client generator

### PR DESCRIPTION
**Pull request by wing328**
Deprecate Flash (ActionScript) client generator

To close https://github.com/OpenAPITools/openapi-generator/issues/7230

cc @ybelenko @OpenAPITools/generator-core-team 

### PR checklist
 
- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [ ] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [ ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [ ] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### CODE REVIEW

**Comment by wing328** [comment]
@ybelenko good catch. I've cleaned up other left-over files due to depreciation.

**Comment by ybelenko** [comment]
@wing328 looks like you did something wrong, this file is deleted docs/generators/README.md completely
Check changes in latest commit again...

File: docs/generators.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The following generators are available:
 * [elm](generators/elm.md)  
 * [erlang-client](generators/erlang-client.md)  
 * [erlang-proper](generators/erlang-proper.md)  
-* [flash-deprecated (deprecated)](generators/flash-deprecated.md)
```
Comment (ybelenko) [inline docs/generators.md]: Looks like `git revert commit` output.

**Comment by wing328** [comment]
@ybelenko Sorry. Just pushed some changes. Please have another look when you've time.

**Comment by ybelenko** [comment]
Did you delete this file on purpose docs/generators/flash-deprecated.md ?

**Comment by ybelenko** [comment]
Seems like you still didn't delete `docs/generators/flash.md` )

**Review by ybelenko — APPROVED** [review]

**Comment by ybelenko** [comment]
@wing328 now I'm fully satisfied, thank you 👍

**Pull request merged**